doc/advanced-configuration/outgoing-connections.md

```diff
@@ -123,7 +123,8 @@ An example configuration can look as follows:
 {outgoing_pools, [
  {rdbms, global, default, [{workers, 5}],
   [{server, {mysql, "localhost", 3306, "mydb", "mim", "mimpass",
-             [{verify, verify_peer}, {cacertfile, "path/to/cacert.pem"}]}}]}
+             [{verify, verify_peer}, {cacertfile, "path/to/cacert.pem"},
+              {server_name_indication, disable}]}}]}
```

Comment:
@DenysGonchar did you mean to change it in `big_tests/test.config` ??

Reply:
no. i just wanted to put in example config. cause that is what people have to provide.